js/index.js

```diff
@@ -38,5 +38,88 @@ const siteContent = {
 };
 
 // Example: Update the img src for the logo
-let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+// let logo = document.getElementById("logo-img");
+// logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+//NAV /////////////
+let navItems = document.querySelectorAll('a');
+//console.log(navItems); 
+
+//HTML Collection - it's array like.
+
+// Going through and grabing the items individually in an array like 
+navItems[0].textContent = siteContent['nav']['nav-item-1'];
```

Comment:
If you wanna push yourself, make `forEach` loop, or just an old school for loop, and assign the values using the loop